model/src/main/java/org/projectnessie/model/Branch.java

```diff
@@ -49,4 +51,15 @@ default void checkName() {
   static Branch of(String name, String hash) {
     return builder().name(name).hash(hash).build();
   }
+
+  /**
+   * Returns a map of metadata properties that describe this branch. Note that these properties
+   * <b>can be added</b> by the server when a {@link Branch} instance is returned.
+   *
+   * @return A map of metadata properties that describe this branch. Note that these properties
+   *     <b>can be added</b> by the server when a {@link Branch} instance is returned.
+   */
+  @Nullable
```

Comment:
Should we maybe default to an empty map and mark this one as not-null?

Reply:
+1 to a non-null empty map default

Reply:
I was thinking about that and the reason why I thought that going with null & not serializing this is so that the python client isn't affected by this change (since the python client doesn't even ask for additional metadata).
We could also use an empty map by default and annotate it with `@JsonInclude(Include.NON_EMPTY)` but the first option seemed to make more sense. thoughts?